### PR TITLE
Qm 001

### PR DESCRIPTION
**Pull request by thea-m**

### CODE REVIEW

**Comment by thea-m** [comment]
This is a PR for @Mersha-Alehegne @MershaMengistie

File: QaranyoMadhaneAlam/QM001.xml [hunk 1]
```diff
@@ -0,0 +1,178 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="QM001" xml:lang="en" type="mss">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>ወንጌል ዘዮሐንስ (wangel za-Yoḥannǝs) Gospel of John</title>
```
Comment (DariaElagina) [inline QaranyoMadhaneAlam/QM001.xml:8]: We mostly used a transliterated Geez title here, so: Wangel za-Yoḥannǝs

File: QaranyoMadhaneAlam/QM001.xml [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <msDesc xml:id="ms">
+                    <msIdentifier>
+                        <repository ref="INS0921QM"/>
+                        <idno>QM001</idno>
+                      <altIdentifier>
+                         <idno>Inventory number: AW 003</idno>
```
Comment (DariaElagina) [inline QaranyoMadhaneAlam/QM001.xml:29]: I am wondering if one could ommit "Inventory number", what do you think @DenisNosnitsin1970 ?

File: QaranyoMadhaneAlam/QM001.xml [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+                                <locus from="3r" to="89va"/>
+                                <title type="complete" ref="LIT1693John#JohnGospel"/>
+                                   <incipit><hi rend="rubric">ቀዳሚሁ፡ ውእቱ፡ ቃል፡ ወውእቱ፡ ቃል፡ ኀ</hi>በ፡ እግዚኣብሔር፡ ሀለወ፡ ወእግዚኣብሔር፡ ውእቱ፡
+                                    ቃል፡ ዝንቱ፡ እምቀዲሙ፡ ሀሎ፡ ኀበ፡ እግዚኣብሔር። ኵሉ፡ ቦቱ፡ ኮነ፡ ወዘእንበሌሁሰ፡ አልቦ፡ ዘኮነ፡፡</incipit>
+                                    <explicit>ወሶበ፡ ተጽሕፈ፡ ኵሉ፡ በበ፡ ኣሓዱ፡ እምኢያገምሮ፡ዓለም፡ ጥቀ፡ መጻሕፍቲሁ፡ ዘእምተጽሕፈ፡ አሜን። ወአሜን። ለይኩን ለይኩን። ወ</explicit>
+                                  </msItem>
+                            <msItem xml:id="ms_i1.3">
+                              <locus target="#89v"/>
+                              <title type="complete" ref="LIT1693John#Postscript"/>
+                              <incipit>መልዓ፡ስብከተ፡ ዮሐንስ</incipit>
```
Comment (DariaElagina) [inline QaranyoMadhaneAlam/QM001.xml:60]: A white space is missing

File: QaranyoMadhaneAlam/QM001.xml [hunk 4]
```diff
@@ -0,0 +1,178 @@
...
+                              <explicit>ወበ፷፡ ዓመተ፡ መንግሥቱ፡ </explicit>
+                                <explicit type="subscriptio">ዝ መጽሐፍ ዘ <persName ref="PRS13313Labbawekrestos" role="owner">ለባዊ ክርስቶስ</persName> ጸሐፊሁ <persName ref="PRS13314Haylamasqal" role="scribe">ኃይለ መስቀል</persName> ወአቡሁ <persName ref="PRS13315TasfaIyyasus"> ተስፋ ኢየሱስ</persName> ወእሙ <persName ref="PRS13316WalattaDawit">ወለተ ዳዊት</persName> ወብእሲቱ <persName ref="PRS13328AbalaMaryam">አባለ ማርያም</persName> ወውልዱ <persName ref="PRS13329Pestqis">ጵስጥቂስ</persName> ዘሠረቆ ወዘፈሐቆ</explicit>
```
Comment (DariaElagina) [inline QaranyoMadhaneAlam/QM001.xml:62]: I would say that this is not a subscriptio, but I could not find the way of encoding this in the GL.

File: QaranyoMadhaneAlam/QM001.xml [hunk 5]
```diff
@@ -0,0 +1,178 @@
...
+                                  <locus target="#90r"/>
+                                  <title type="complete" ref="LIT1693John#TituliJohn"><hi rend="rubric">ዘዮሐንስ፡ ፳፡ አርእስት፡</hi></title>
+                                  <incipit>በእንተ፡ ከብካብ፡ ዘቃና</incipit>
+                                  <explicit>ተፈጸመ፡ ዘዮሐንስ፡ አርእስት፡</explicit>
+                                    </msItem>
+                                    <msItem xml:id="ms_i1.5">
+                                      <locus from="90rb" to="92va"/>
+                                      <title type="complete" ref="LIT1693John#BiographyJohn"></title>
+                                      <incipit><hi rend="rubric">በስመ፡ አብ፡ ወወልድ፡ ወመንፈስ፡</hi>ቅዱስ ፩ በመለኮት፡ ወይሤለስ፡ በመለኮት፡ ወይሤለስ፡ በአካላት፡ ለዓለም፡ ዓለም፡ አሜን።</incipit>
+                                      <explicit>ሎቱ፡ ስብሐት፡ ክብር፡ ለአብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ ይእዜኒ፡ወዘልፈኒ፡ ወለዓለመ፡ ዓለም፡ አሜን።</explicit>
```
Comment (DariaElagina) [inline QaranyoMadhaneAlam/QM001.xml:74]: a white space is missing in ይእዜኒ፡ወዘልፈኒ፡

File: QaranyoMadhaneAlam/QM001.xml [hunk 6]
```diff
@@ -0,0 +1,178 @@
...
+                                  </support>
+                                  <extent>
+                                     <measure unit="leaf">96</measure>
+                                     <measure unit="leaf" type="blank">1</measure>
+                                     <locus target="#95v"/>
+                                    A + <measure unit="quire">9</measure> + B
+                                     </extent>
+                                  <foliation/>
+                                  <condition key="other">the text-block is well preserved but its binding was repaired. The front board was broken into
+                                    two pieces and was repaired by sewed together in three spots. One of the guard leaves is used to reinforce the spine</condition>
```
Comment (DariaElagina) [inline QaranyoMadhaneAlam/QM001.xml:93]: by sewing?

File: QaranyoMadhaneAlam/QM001.xml [hunk 7]
```diff
@@ -0,0 +1,178 @@
...
+                                       <q xml:lang="gez"></q>
+                                    </item>
+                                    <item xml:id="a3">
+                                    <locus target="#94r"></locus>
+                                     <desc type="Unclear">unknown text</desc>
+                                         <q xml:lang="gez"></q>
+                                      </item>
+                                      <item xml:id="a4">
+                                      <locus target="#95r"></locus>
+                                       <desc type="CalendaricNote">not standard</desc>
```
Comment (DariaElagina) [inline QaranyoMadhaneAlam/QM001.xml:123]: I would write it out like: Non-standard calendaric note

File: QaranyoMadhaneAlam/QM001.xml (hunk 2 above)
Comment (MershaMengistie) [reply QaranyoMadhaneAlam/QM001.xml:29]: I suggest it should remain there as it tells the inventory number not the shelfmark.

**Comment by MershaMengistie** [comment]
@DariaElagina: can my encoded ms be merged?

**Review by DariaElagina — APPROVED** [review]

**Pull request merged**